src/dftbp/dftbplus/main.F90

```diff
@@ -6056,17 +6057,20 @@ end subroutine unconstrainLatticeVectors
 
 
   !> Returns the coordinates for the next Hessian calculation step.
-  subroutine getNextDerivStep(derivDriver, derivs, indMovedAtoms, coord, tGeomEnd)
+  subroutine getNextDerivStep(derivDriver, derivs, indMovedAtoms, indCompAtoms, coord, tGeomEnd)
 
     !> Driver for the finite difference second derivatives
     type(TNumDerivs), intent(inout) :: derivDriver
 
     !> first derivatives of energy at the current coordinates
     real(dp), intent(in) :: derivs(:,:)
 
-    !> moving atoms
+    !> indices of moving atoms
     integer, intent(in) :: indMovedAtoms(:)
 
+    !> indices of computed atoms
+    integer, intent(in) :: indCompAtoms(:)
+
     !> atomic coordinates
     real(dp), intent(out) :: coord(:,:)
```

Comment:
```suggestion
    real(dp), intent(inout) :: coord(:,:)
```
Must be `intent(inout)` as otherwise it may loose its value at entry.

Reply:
I did this. 
Concerning squash I read it is possible to squash all commit when merging a PR even within the gitHub interface. From my side I'd need to create  another branch and place there all commits and do another PR.

Reply:
Concerning the script, I was looking at something pythonic like here:
https://vsoch.github.io/lessons/sherlock-jobs/

Ok, we can do something along these lines if you like, but it is not clear if we gain a lot.
Clearly the job script (slurm or whatver) needs to contains preset with correct user options. The dftb_in also need to be prepopulated with correct inputs. 
So, in practice, the script simply needs to create the run folders, copy files there and start the jobs. Manipulations are minimal. 
Can phonopy start parallel jobs of the dynamical matrix ?